lib/auth/SpacedeckAuthProvider.js

```diff
@@ -23,11 +34,7 @@ class SpacedeckAuthProvider {
         // If the token is not authorized, the Spacedeck API still returns a payload; therefore we have
         // to check if there _is_ any payload, and if this payload _does_not_contain_ an 'error' key.
         // If these contiditions are matched, then we can assume the token to be momentarily valid.
-        if (tokenValidationCall && !tokenValidationCall.error) {
-            return true;
-        } else {
-            return false;
-        }
+        return !!(tokenValidationCall && !tokenValidationCall.error);
```

Comment:
No need for the `!!( )` -- the inside is already going to be a `boolean` at all times.